app/assets/v2/js/pages/bounty_details.js

```diff
@@ -430,6 +430,18 @@ var show_interest_modal = function() {
         var has_question = event.target[0].value;
         var issue_message = event.target[2].value;
 
+        var agree_precedence = event.target[3].checked;
+        var agree_not_to_abandon = event.target[4].checked;
+
+        if (!agree_precedence) {
+          alert('You must agree to the precedence clause.');
```

Comment:
Would it make sense to reuse our our custom alert like [here](https://github.com/gitcoinco/web/blob/master/app/assets/v2/js/pages/dashboard.js#L470)

`_alert({message: 'You must agree to the precedence clause.'}, 'error');`